counter/src/Main.gren

```diff
@@ -31,14 +35,19 @@ update msg model =
 
 
 view : Model -> Html Msg
-view model = 
-    Html.div []
-        [ Html.span 
-            [ Attribute.id "count" ]
-            [ Html.text <| String.fromInt model ]
+view model =
+    Html.div
+        []
```

Comment:
When the first argument is a simple expression, like `[]` or an `Int` or a `String`, I think it looks nicer when being placed on the same line as the function call. I believe that's also how `elm-format` works.

Reply:
Yes, elm-format allows function calls to be one of (with the added rule that if any argument spans multiple lines, then it is not allowed to be on the same line as anything else):
- function and all arguments on the same line
- function and first argument on the same line, all other arguments on individual lines
- everything on separate lines
 
It's a rule that I feel like isn't very elegant, and I guess it's been fine in Elm, but I'm not 100% sure it always makes sense.

We could use that same rule for gren, or consider some different kind of rule for this.